printer_mods/gauravmm/crazy_mozzie_cw2/README.md

```diff
@@ -0,0 +1,14 @@
+# Stealthburner + Clockwork2 Toolhead for HF Crazy Mozzie
+
+This is a a toolhead mount for the HF Crazy Mozzie (or other Mosquito clones), compatible with the final release of Stealthburner and Clockwork 2. [Design source.](https://a360.co/3CtpGPo)
```

Comment:
Stealthburner isn't on final release yet. AFAIK it's still release candidate 1.

Reply:
Done!

Reply:
Can you please specify stealthburner RC 1 in case there are breaking changes in a future version?

Reply:
Done!